<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormularyFrame</class>
 <widget class="QFrame" name="FormularyFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>913</width>
    <height>640</height>
   </rect>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Plain</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Medication search</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="nameLabel">
          <property name="text">
           <string>MEDICATION/BARCODE</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="nameField">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>250</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QCheckBox" name="activeCheckbox">
          <property name="text">
           <string>Active</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="resetButton">
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="searchButton">
       <property name="text">
        <string>Search</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>60</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="newMedicationButton">
       <property name="text">
        <string>New Medication</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>60</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="newStockButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Add Stock</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="modifyButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Modify Medication</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="resultTable">
     <property name="contextMenuPolicy">
      <enum>Qt::ActionsContextMenu</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="columnCount">
      <number>6</number>
     </property>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>100</number>
     </attribute>
     <column>
      <property name="text">
       <string>MEDICATION</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>NDC</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>FORM</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>STRENGTH</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>UNIT SIZE</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>STOCK</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
  <action name="newMedicationAction">
   <property name="text">
    <string>New Medication</string>
   </property>
   <property name="toolTip">
    <string>Add a new medication.</string>
   </property>
  </action>
  <action name="newStockAction">
   <property name="text">
    <string>Add Stock</string>
   </property>
   <property name="toolTip">
    <string>Add new stock of a medication.</string>
   </property>
  </action>
  <action name="modifyAction">
   <property name="text">
    <string>Modify Medication</string>
   </property>
   <property name="toolTip">
    <string>Modify medication information.</string>
   </property>
  </action>
  <action name="toggleAction">
   <property name="text">
    <string>Toggle Active</string>
   </property>
   <property name="toolTip">
    <string>Toggle active/inactive</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>nameField</sender>
   <signal>returnPressed()</signal>
   <receiver>searchButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>168</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>129</x>
     <y>78</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>resetButton</sender>
   <signal>clicked()</signal>
   <receiver>nameField</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>168</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>newStockButton</sender>
   <signal>clicked()</signal>
   <receiver>newStockAction</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>modifyButton</sender>
   <signal>clicked()</signal>
   <receiver>modifyAction</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>854</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>newMedicationButton</sender>
   <signal>clicked()</signal>
   <receiver>newMedicationAction</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>280</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
